### PR TITLE
Fix aws result formatter

### PR DESCRIPTION
**Pull request by m4dcoder**

### CODE REVIEW

**Comment by estee-tew** [comment]
https://stackstorm.slack.com/archives/stackstorm/p1454640331005179

File: packs/aws/actions/lib/ec2parsers.py [hunk 1]
```diff
@@ -35,6 +130,8 @@ def selector(self, output):
             return self.parseInstance(output)
         elif isinstance(output, boto.ec2.volume.Volume):
             return self.parseVolume(output)
+        elif isinstance(output, boto.ec2.blockdevicemapping.BlockDeviceType):
```
Comment (manasdk) [inline packs/aws/actions/lib/ec2parsers.py]: since BlockDeviceType is an object(https://github.com/boto/boto/blob/d9e5cfe900e1a58717e393c76a6e3580305f217a/boto/ec2/blockdevicemapping.py#L25) can we use some generic mechanism to parse all python objects.

Writing out FieldList like https://github.com/StackStorm/st2contrib/pull/386/files#diff-417c109ec13ece08b2872bd616c2e1f3R17 leads to too much special casing unfortunately.

Comment (m4dcoder) [inline packs/aws/actions/lib/ec2parsers.py]: Let's move the parser rework as a separate PR for now so we don't block here.

Comment (manasdk) [inline packs/aws/actions/lib/ec2parsers.py]: Created issue https://github.com/StackStorm/st2contrib/issues/387 for tracking. Since marshalling and unmarshalling object properly is my pet-peeve I might take a crack at it :P

**Comment by estee-tew** [comment]
https://stackstorm.slack.com/archives/stackstorm/p1455084860006236
